Example/ViewController.swift

```diff
@@ -343,24 +299,25 @@ extension ViewController {
                 return duration != nil
             }
             .map { return ($0.0.0, $0.1!) } //unwrap current time & duration
-            .map(setSliderPosition)
+            .map({ [unowned self] position in
```

Comment:
I believe you should keep weak reference here.<br/>
Even if it is unlikely in your configuration, this closure could potentially be called with released viewController

Reply:
hum, if viewController is released, disposebag is called just before no ?